src/components/FeatureViews/Accounts/Accounts.js

```diff
@@ -300,13 +299,14 @@ class Accounts extends Component {
                                 indicatorColor="primary"
                                 onChange={this.handleChange}
                                 textColor="primary"
+                                scrollButtons="off"
                                 value={this.state.tabIndex}
                                 variant="scrollable">
-                                <Tab label="ALL" classes={styles("").first_tab}/>
-                                <Tab label="INSTRUCTORS" />
-                                <Tab label="STUDENTS" />
-                                <Tab label="RECEPTIONIST" />
-                                <Tab label="PARENTS" />
+                                <Tab style={{minWidth:130}} label="ALL" classes={styles("").first_tab}/>
```

Comment:
Instead of c/ping the minWidth for each tag, make this a CSS class and apply this to all the tags. Otherwise updating all the tab width becomes a pain.